app/config.go

```diff
@@ -7,35 +7,38 @@
 package app
 
 import (
+	"fmt"
 	"strings"
 
 	"cosmossdk.io/math"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/evmos/evmos/v20/app/eips"
-	"github.com/evmos/evmos/v20/utils"
-	"github.com/evmos/evmos/v20/x/evm/core/vm"
 	evmtypes "github.com/evmos/evmos/v20/x/evm/types"
 )
 
+// EvmosOptionsFn defines a function type for setting app options specifically for
+// the Evmos app. The function should receive the chainID and return an error if
+// any.
+type EvmosOptionsFn func(string) error
+
+// NoOpEvmosOptions is a no-op function that can be used when the app does not
+// need any specific configuration.
+func NoOpEvmosOptions(_ string) error {
+	return nil
+}
+
 var sealed = false
 
-// InitializeAppConfiguration allows to setup the global configuration
-// for the Evmos EVM.
-func InitializeAppConfiguration(chainID string) error {
+// EvmosAppOptions allows to setup the global configuration
+// for the Evmos chain.
+func EvmosAppOptions(chainID string) error {
 	if sealed {
 		return nil
 	}
 
-	// When calling any CLI command, it creates a tempApp inside RootCmdHandler with an empty chainID.
-	if chainID == "" {
-		return nil
-	}
```

Comment:
why is it not necessary to handle the empty chain ID anymore?

Reply:
Because it was used for the creation of the tmp app in the commands. Now we can directly use the NoOpEvmosOptions which is more clear